src/components/tag/TagList.tsx

```diff
@@ -0,0 +1,40 @@
+import styled from "@emotion/styled";
+import { HTMLAttributes } from "react";
+import TagItem, { TagItemProps } from "components/tag/TagItem";
+import React, { useState } from "react";
+
+interface Props extends HTMLAttributes<HTMLDivElement> {
+  deletable?: boolean;
+}
+
+const TagList: React.FC<Props> = ({ children, deletable = false, ...props }: Props) => {
+  const [tagItems, setTagItems] = useState(children);
+
+  const handleTagItemClick = (index: number) => {
+    const newTagItems = React.Children.toArray(tagItems).filter((_, idx) => idx !== index);
+    setTagItems(newTagItems);
+  };
+
+  const tagItemsWithOnClick = React.Children.map(tagItems, (child, index) => {
+    if (deletable && React.isValidElement<TagItemProps>(child)) {
+      return React.cloneElement(child, {
+        onClick: () => handleTagItemClick(index),
+      });
+    }
+    return child;
+  });
+
+  return <EmotionWrapper {...props}>{tagItemsWithOnClick}</EmotionWrapper>;
+};
+
+export default TagList;
+
+const EmotionWrapper = styled.div<Props>`
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 5px;
+  align-items: flex-start;
+`;
+
+export { TagItem };
```

Comment:
아래와 같이 변경 하면 사용처에서 <TagList.Item> 과 같이 사용 할 수 있습니다!

```suggestion
TagList.Item = TagItem
```

Reply:
또는 네이밍 변경 제안 드린대로, 아래와 같이 변경하는게 낫지 싶긴 합니다.
```suggestion
Tags.Item = TagItem
```